rpcs3/Input/ds4_pad_handler.cpp

```diff
@@ -906,6 +922,28 @@ void ds4_pad_handler::apply_pad_data(const std::shared_ptr<PadDevice>& device, c
 		newBlinkData = true;
 	}
 
+	// Use LEDs to indicate battery level
+	// TODO: Implement a setting to enable this. The following code works and correctly indicates the current battery level by changing the LED color.
+	if (false) 
```

Comment:
This will probably need a new value in pad_config near colorR, colorG and colorB

Reply:
I agree, this is a way I intend to move forward if this feature is to be implemented fully.